purchases/src/main/kotlin/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -266,6 +266,7 @@ internal class HTTPClient(
             "X-Platform-Flavor" to appConfig.platformInfo.flavor,
             "X-Platform-Flavor-Version" to appConfig.platformInfo.version,
             "X-Platform-Version" to Build.VERSION.SDK_INT.toString(),
+            "X-Platform-Device" to Build.MODEL,
```

Comment:
Would adding [`Build.BRAND`](https://developer.android.com/reference/android/os/Build#BRAND) be useful here as well? Otherwise we might only see something like "SM-S921E" without knowing it's a Samsung (Galaxy S24 in this case). 

(Samsung's model numbers are quite recognizable, but others might be less so.)

Reply:
Hmm could be useful yeah... I'm just reticent to send it in the same header, so the backend doesn't need to process the string. iOS wouldn't need a similar header, but we can potentially send it in case it's useful in the future. Will add it as a separate header 👍